utils/display.go

```diff
@@ -960,6 +965,70 @@ func extendedFunctionsTable(functions []*meroxa.Function) string {
 	return subTable
 }
 
+// BuildsTable displays multiple build records for future listing per app.
+func BuildsTable(builds []*meroxa.Build, hideHeaders bool) string {
```

Comment:
I just noticed this is not being used anywhere. Will remove in https://github.com/meroxa/cli/pull/386 as part of a refactor I'm doing.

Reply:
Done in https://github.com/meroxa/cli/pull/386/commits/38ba1a0196473cb8ba0ad950e7f054803eda3b6b.